config.json

```diff
@@ -5,7 +5,7 @@
     "winrm_port": 5985,
     "repository": "virtio-win/kvm-guest-drivers-windows",
     "studio_username": "Administrator",
-    "studio_password": "Qum5net.",
+    "studio_password": "Assentor01",
```

Comment:
I prefered don't touch value in `config.json` because we already have a lot of deployments that use default password. For new deployments lets set it in `override.json`. Maybe add question in AutoHCK-Installer.